cmd/kube-rbac-proxy/app/kube-rbac-proxy.go

```diff
@@ -451,7 +451,7 @@ func Run(cfg *completedProxyRunOptions) error {
 				return srv.Serve(tlsListener)
 			}, func(err error) {
 				if err := srv.Shutdown(context.Background()); err != nil {
-					klog.Errorf("failed to gracefully shutdown server: %w", err)
```

Comment:
This does not appear to be a bump commit at all? So many test changes, too.

Reply:
`klog.Errorf` -> `klog.ErrorS`, it was necessary after the bump.

I moved the validation from the completed config to the options, like in apiserver, such that I don't need to add the disabled options to the config.

Reply:
Why? `klog.Errorf` is still present.

The test changes below don't belong to the bump commit.